main/zoe/guide/contracts/second-price-auction.md

```diff
@@ -2,117 +2,129 @@
 
-const { outcome: aliceOfferResult, payout: alicePayoutP } = await E(zoe).offer(
-  aliceInvite,
-  aliceProposal,
-  alicePayments,
-);
+const aliceSeat = await E(zoe).offer(creatorInvitation, aliceProposal, alicePayments);
+const invitationMaker = await E(aliceSeat).getOfferResult();
+const bobInvitation = E(invitationMaker).makeBidInvitation();
 ```
 
-Now Alice can spread her invite far and wide and see if
-there are any bidders. Let's say that Bob gets the invite and
-wants to see if it is the kind of contract that he wants to join. He
-can check that the installationHandle installed is the swap he is expecting. He can also check that the item up for sale is the kind that he wants by comparing the issuers.
+Now Alice can share the counterparty invitation with her friends and see if there are
+any bidders. Let's say that Bob gets an invitation and wants to verify that it is an
+offer he's interested in. He can check that the installation is the standard auction
+he expects. He can also check that the item up for sale is what he wants by comparing
+the issuers.
 
 ```js
-// Bob collects information about the swap
-const inviteIssuer = zoe.getInviteIssuer();
-const bobExclusiveInvite = await inviteIssuer.claim(bobInvite);
-const bobInviteValue = inviteIssuer.getAmountOf(bobExclusiveInvite)
-  .value[0];
+const invitationIssuer = await E(zoe).getInvitationIssuer();
+const bobExclusiveInvitation = await invitationIssuer.claim(bobInvitation);
 
 const {
-  installationHandle: bobInstallationId,
-  issuerKeywordRecord: bobIssuers,
-} = zoe.getInstance(bobInviteValue.instanceHandle);
+  installation: bobInstallation,
+  instance,
+} = await E(zoe).getInvitationDetails(bobExclusiveInvitation);
+const bobIssuers = await E(zoe).getIssuers(instance);
 
-// Bob checks the information is what he expects
-assert(bobInstallationId === installationHandle, details`wrong installation`);
+assert(bobInstallation === secondPriceAuctionInstallation, details`wrong installation`);
 assert(bobIssuers.Asset === moolaIssuer, details`wrong issuer`);
 ```
 
-Bob decides to join the contract and
-makes an offer:
+Bob decides to join the contract and makes an offer:
 
 ```js
 const bobProposal = harden({
-  give: { Bid: simoleans(11) },
-  want: { Asset: moola(1) },
+  give: { Bid: simoleanAmountMath.make(11) },
+  want: { Asset: moolaAmountMath.make(1) },
 });
 
 const bobPayments = { Bid: bobSimoleanPayment };
 
-// Bob escrows with Zoe and bids
-const { outcome: bobOfferResult, payout: bobPayoutP } = await E(zoe).offer(
-  bobExclusiveInvite,
-  bobProposal,
-  bobPayments,
-);
-
+const bobSeat = await E(zoe).offer(bobExclusiveInvitation, bobProposal, bobPayments);
 ```
 
 Since multiple parties may want to participate in the auction, let's say that Carol and Dave also decide to bid in the same way
-as Bob- Carol bidding 7 simoleans, and Dave bidding 5 simoleans.
+as Bob&mdash;Carol bidding 7 simoleans, and Dave bidding 5 simoleans.
```

Comment:
huh, why not use the character itself?

Reply:
hyphens and m-dashes are different characters. I guess I've been editing a fair amount of html recently.